clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -557,6 +558,7 @@ private void handleListOffsetResponse(Map<TopicPartition, Long> timestampsToSear
                                           ClientResponse clientResponse,
                                           RequestFuture<Map<TopicPartition, OffsetAndTimestamp>> future) {
         ListOffsetResponse lor = new ListOffsetResponse(clientResponse.responseBody());
+        log.trace("Received ListOffsetResponse {}", lor);
```

Comment:
Can we combine the trace message above (which prints the node that received the message) and this one so that we know which response came from which node? Maybe the easy way to do this is to move the parsing of the `ListOffsetResponse` into the `onSuccess` call in `sendListOffsetRequest`.